flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkDistributionMode.java

```diff
@@ -118,9 +118,15 @@ public void before() throws IOException {
   public void testShuffleByPartitionWithSchema() throws Exception {
     testWriteRow(parallelism, SimpleDataUtil.FLINK_SCHEMA, DistributionMode.HASH);
     if (partitioned) {
-      assertThat(partitionFiles("aaa")).isEqualTo(1);
-      assertThat(partitionFiles("bbb")).isEqualTo(1);
-      assertThat(partitionFiles("ccc")).isEqualTo(1);
+      assertThat(partitionFiles("aaa"))
+          .as("There should be only 1 data file in partition 'aaa'")
+          .isEqualTo(1);
+      assertThat(partitionFiles("bbb"))
+          .as("There should be only 1 data file in partition 'bbb'")
+          .isEqualTo(1);
+      assertThat(partitionFiles("ccc"))
+          .as("There should be only 1 data file in partition 'ccc'")
+          .isEqualTo(1);
```

Comment:
Do the comments in this file add any value above the ones we can read from the code?

Reply:
I can't answer on this question because I'm not up-to-date with assertj. I copied it from this PR: https://github.com/apache/iceberg/pull/10179/files#diff-42c31c3c7e3e8287ea451196f6c0b4d0a47aa931391df70369a87ed3d19a1452R221 @rodmeneses copied this code for SinkV2 and (I assume) forgot to backport it to the original source of the code. We can synchronize it in the opposite direction (remove these explanations from duplicated places) if we find out that it doesn't add a value.

Reply:
Ok.. I'm fine in both ways, and I agree with the sync effort.
@rodmeneses: What is the history of this difference? Is this intentional?